pkg/controllers/cstorvolumeconfig/controller_test.go

```diff
@@ -71,7 +71,6 @@ type fixture struct {
 
 	// Actions expected to happen on the client. Objects from here are also
 	// preloaded into NewSimpleFake.
-	actions        []core.Action
```

Comment:
Is linter shown this field as unused?

Reply:
yes!

Reply:
But we are using them as we can see [here](https://github.com/openebs/cstor-operators/pull/301/checks?check_run_id=2500796432) test cases are failing.

Reply:
We can undo the changes done to `actions`